src/main.rs

```diff
@@ -50,6 +53,7 @@ Options:
     --charset CHARSET       Set the character set to use in output. Valid
                             values: utf8, ascii [default: utf8]
     -f, --format FORMAT     Format string for printing dependencies
+    -j, --json              Print a JSON representation of the tree
```

Comment:
I think it'd make sense to match the `--output-format` flag that Cargo uses, and make the JSON representation compatible with that.